### ISSUE TITLE
login is insecure

### ISSUE THREAD

**Issue by quozl** [sugarlabs/aslo#5]
activities.sugarlabs.org offers a login through insecure http; it should first redirect to https.

**Comment by jatindhankhar** [comment]
Yes I even noticed the same. Also registration form is not a pleasant thing to look at. Needs to be re-designed as well.
![screenshot from 2017-03-30 16-02-45](https://cloud.githubusercontent.com/assets/2412081/24520494/4eb082ac-15a6-11e7-89a1-f34094632927.png)

**Comment by quozl** [comment]
must admit it was firefox what told me; i had not noticed or worried before.

**Comment by iamutkarshtiwari** [comment]
@jatindhankhar This PR should fix that issue - https://github.com/sugarlabs/aslo/pull/10

**Comment by quozl** [comment]
thanks.
